docs-website/sphinx/apidocs/sdk-v2/entities.rst

```diff
@@ -0,0 +1,44 @@
+Entities
+=======
+
+The DataHub SDK provides a set of entities that can be used to interact with DataHub's metadata.
+
+Dataset
+-------
+
+.. automodule:: datahub.sdk.dataset
```

Comment:
on the dataset entity init - what's up with the `**` stuff? also, ideally the user should know what those type aliases actually allow for? 
<img width="724" alt="image" src="https://github.com/user-attachments/assets/1889e93a-487a-4957-83c1-6b1a769ee53b" />

Reply:
added parsing for this